doc/nrf/known_issues.rst

```diff
@@ -1357,90 +1387,110 @@ OCT-2501: Charging over seven hours results in error
 
   **Workaround:** To start the charging again, turn the nRF5340 Audio DK off and then on again.
 
-.. rst-class:: v2-3-0
-
-OCT-2472: Headset fault upon gateway reset in the bidirectional stream mode
-  The headset may react with a usage fault when the :kconfig:option:`CONFIG_STREAM_BIDIRECTIONAL` application Kconfig option is set to ``y`` and the gateway is reset during a stream.
-  This issue is under investigation.
```

Comment:
Are you sure you want to remove issues that we have had before?

Reply:
I think we remove this issue accidently, since it happens in the previous version but fixed now.

Reply:
What about the other removed ones below? Same thing?